src/js/components/RangeInput/stories/Modes.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Box, Grommet, RangeInput } from 'grommet';
+
+import { Volume } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customThemeRangeInput = {
```

Comment:
Thinking we want to focus on the `color` prop aspect. Consider removing the custom theme so that developers are able to focus on the relevant aspect of the example.

Reply:
Sure, will do